.github/pull_request_template.md

```
@@ -1,29 +1,29 @@
 # Overview
 
-### What is the feature/fix?
+### What is the objective?
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)